test/browser/webpack-helper.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+  @param {webpack-context} r - all files in the context will be loaded.
+*/
+function importAll(r) {
+  r.keys().forEach(r);
+}
+
+// Import all libraries from lib directory.
+importAll(require.context(`${__dirname}/../../lib`, true, /\.js$/));
+
+// Require external libraries.
+let libs = {
+  objectGraph: require('object-graph-js').ObjectGraph,
```

Comment:
nit: If we're exposing a class name, let's use `ObjectGraph` instead of `objectGraph`.